server/services/prison/PrisonRegisterService.ts

```diff
@@ -13,12 +13,19 @@ export default class PrisonRegisterService {
   }
 
   async getActivePrisonsFromPrisonRegister(): Promise<Array<Prison>> {
+    let activePrisons: Array<PrisonDto>
```

Comment:
At the moment all I've done is change this public method (`getActivePrisonsFromPrisonRegister`) to use the new type from the cache (`PrisonDto`) and map it into the return type `Prison` (which is our simpler representation containing just an id and name)
At the moment this method is not used as I've not changed the calling code - the calling code still calls the methods that use the static json file.
This is all precursor work to making that switch over